fiftyone/utils/sam3.py

```
@@ -30,6 +33,61 @@
 logger = logging.getLogger(__name__)
 
 
+def _ensure_sam3_bpe_vocab_path(
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_ensure_sam3_bpe_vocab_path`

(ANN202)